hooks/hipchat.js

```diff
@@ -0,0 +1,48 @@
+
+const Hook = require("../hook");
+
+const MXControlEvent = require("../events/mxcontrol");
+const MXControlHandler = require("../handlers/mxcontrol");
+
+const parseHipchatTrigger = (trigger) => {
+
+    const simpleTrigger = simplifyHipchatTrigger(trigger);
+
+    const events = [];
+
+    if (simpleTrigger.event == "room_message") {
+        const commandArgs = simpleTrigger.message.split(" ");
+
+        if (commandArgs[0] == "/mxcontrol")
+            events.push(new MXControlEvent(
+                trigger,
+                commandArgs[1], //action
+                commandArgs[2], //targetType
+                commandArgs[3], //target
+                commandArgs[4]  //size
+            ));
+    }
+
+    return events;
+};
+
+const simplifyHipchatTrigger = (trigger) => {
```

Comment:
can destructure instead of having this function

Reply:
Will do when I come back